avaframe/ana4Stats/getStats.py

```diff
@@ -73,11 +73,26 @@ def extractMaxValues(inputDir, cfgMain, avaDir, nameScenario=''):
     peakFiles = fU.makeSimDict(inputDir, varPar, avaDir)
     nSims = len(peakFiles['simName'])
 
-    # initialize dictionary to save values
+    # load parameter variation values to check if they include default value
+    varParValues = cfgMain['PARAMETERVAR']['varParValues']
+    if ':' in varParValues:
+        itemsInput = varParValues.split(':')
+        itemsRaw = np.linspace(float(itemsInput[0]), float(itemsInput[1]), int(itemsInput[2]))
+    else:
+        itemsRaw = varParValues.split('_')
+        itemsRaw = np.array(itemsRaw,dtype=float)
+    flagValue = False
+    if float(cfgMain['DEFVALUES'][cfgMain['PARAMETERVAR']['varPar']]) in itemsRaw:
+        flagValue = True
+
+    # initialize dictionary to save values, if default value not in parameter variation, exclude this value
```

Comment:
I suggest moving this to a general (i.e. utils) function as this might be used in more places in the code? 
I.e. a function that gets the cfgParameter that needs to be split and returns the split item...

Reply:
I made a function in com1DFA now that is called readVarPar and is also used in getStats.py